### ISSUE TITLE
Verify / Update all python assets for Python3 Compatibility 

### ISSUE THREAD

**Issue by jaketf** [GoogleCloudPlatform/professional-services#293]
This issue is to track updates to existing assets to keep them from going stale in [coming python2 deprecatation](https://pythonclock.org/).

Getting list of python assets
```bash
find ./professional-services -name *.py -type f | awk '{ FS = "/"}; {print $3 "." $4}' | uniq
```
- [ ] tools.dns-sync
- [x] tools.bigquery-query-plan-exporter
- [x] tools.bqpipeline
- [x] tools.ml-dataprep
- [x] tools.asset-inventory
- [ ] tools.gsuite-exporter
- [ ] tools.gcs-bucket-mover
- [x] tools.agile-machine-learning-api
- [ ] tools.labelmaker
- [x] tools.hive-bigquery
- [x] tools.kunskap
- [ ] tools.cloudconnect
- [ ] tools.site-verification-group-sync
- [ ] tools.gce-quota-sync
- [ ] tools.cloud-vision-utils
- [ ] examples.cloudml-bee-health-detection
- [ ] examples.cloudml-collaborative-filtering
- [ ] examples.dataproc-persistent-history-server
- [ ] examples.e2e-home-appliance-status-monitoring
- [x] examples.dataflow-python-examples
- [x] examples.cloud-composer-examples
- [x] examples.cloudml-churn-prediction
- [ ] examples.bigquery-row-access-groups
- [ ] examples.qaoa
- [ ] examples.cloudml-sklearn-pipeline
- [x] examples.dataflow-data-generator
- [ ] examples.gcf-pubsub-vm-delete-event-handler
- [ ] examples.cloudml-fraud-detection
- [ ] examples.bigquery-cross-project-slot-monitoring
- [ ] examples.kubeflow-fairing-example
- [ ] examples.gce-to-adminsdk
- [ ] examples.kubeflow-pipelines-sentiment-analysis
- [ ] examples.dlp
- [ ] examples.tensorflow-profiling-examples
- [ ] examples.cloudml-energy-price-forecasting
- [ ] examples.cloudsql-custom-metric
- [x] examples.python-cicd-with-cloudbuilder
- [x] examples.cryptorealtime
- [ ] examples.cloudml-sentiment-analysis
- [ ] helpers.sort_lists.py
